### PR TITLE
fozzie@10.9.0 - Fix failing tests

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Fixed
- Failing tests.
- Linting errors.
  - Including renaming keyframe animation names to kebab-case.

### CODE REVIEW

**Comment by xander-marjoram** [comment]
As far as I'm aware, the animation names shouldn't be used outside of this package, so renaming them shouldn't be an issue?

**Review by jamieomaguire — DISMISSED** [review]

File: packages/tools/fozzie/src/test/scss/scss-setup.spec.js [hunk 1]
```diff
@@ -11,13 +11,15 @@ const sassTrue = require('sass-true');
 const glob = require('glob');
 
 describe('Sass', () => {
-    // All Scss unit tests should follow this naming convention
-    const testFileGlob = '**/*.spec.scss';
-    const testFilePathGlob = path.resolve(process.cwd(), testFileGlob);
+    it('Find and run tests', () => {
```
Comment (jamieomaguire) [inline packages/tools/fozzie/src/test/scss/scss-setup.spec.js]: nice change - not sure why I didn't do this before

Comment (jamieomaguire) [reply packages/tools/fozzie/src/test/scss/scss-setup.spec.js]: NIT: can we rename to 'Find and run SCSS unit tests'?

Comment (kevinrodrigues) [reply packages/tools/fozzie/src/test/scss/scss-setup.spec.js]: If you change the main describe to scss then you won't need to add it here again.

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/tools/fozzie/src/test/scss/scss-setup.spec.js (hunk 1 above)
Comment (jamieomaguire) [reply packages/tools/fozzie/src/test/scss/scss-setup.spec.js]: So `describe('SCSS')` then `it('Find and run unit tests')`? Sounds good to me if so!

Comment (kevinrodrigues) [reply packages/tools/fozzie/src/test/scss/scss-setup.spec.js]: Yeah, so it will read:

`SCSS >  should find and run unit tests`

**Review by kevinrodrigues — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**